modules/aleph/tests/AlephMaterialUnit.test

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/**
+ * @file
+ * Contains the Aleph Material specific unit tests.
+ */
+
+use Drupal\aleph\Aleph\AlephMaterial;
+use Drupal\ding_test\DingUnitTestBase;
+
+/**
+ * Class AlephMaterialUnitTest.
+ */
+class AlephMaterialUnitTest extends DingUnitTestBase {
```

Comment:
This test is failing, but I think it is mostly because of some issues i've fixed in bbs-sal - I'll get it merged to bbs-master and then we'll rebase your branch.